website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md

```diff
@@ -0,0 +1,259 @@
+---
+authors: 
+  - wwayne
+  - icycodes
+  - gyxlucy
+
+tags: [tech design]
+---
+# HTTP Streaming Back-Pressure and Cancellation in Code Completion
+
+![back-pressure](./intro.png)
```

Comment:
```suggestion
![Introduction](./intro.png)
```

Alt string is mostly used for search engine understanding the content of image, so it should be semantic relevant to image.

Reply:
Please also fix other cases.